events/include/ignition/common/KeyEvent.hh

```diff
@@ -40,6 +40,10 @@ namespace ignition
       /// \brief Constructor.
       public: KeyEvent();
 
+      /// \brief Copy constructor.
+      /// \param[in] _other Other key event
+      public: KeyEvent(const KeyEvent &_other);
```

Comment:
Let's go ahead and add move constructor and move assignment to be rule-of-five compliant.